app/jobs/split_pdf_job.rb

```diff
@@ -22,12 +22,14 @@ def perform(exam_template, path, original_filename=nil, current_user=nil)
     m_logger = MarkusLogger.instance
     begin
       # Create directory for files whose QR code couldn't be parsed
-      error_dir = File.join(exam_template.base_path, 'error')
-      raw_dir = File.join(exam_template.base_path, 'raw')
+      error_dir = File.join(MarkusConfigurator.markus_exam_template_dir,
+                            exam_template.assignment.short_identifier, 'error')
```

Comment:
This change seems like it goes in the opposite direction. You seem to have made the proper change to `base_path` already.